README.md

```diff
@@ -13,8 +13,8 @@ Add `:ueberauth` and `:ueberauth_cognito` to your `mix.exs`:
 defp deps do
   [
     # ...
-    {:ueberauth, "~> 0.1"},
-    {:ueberauth_cognito, git: "https://github.com/mbta/ueberauth_cognito.git"}
+    {:ueberauth, "~> 0.6"},
+    {:ueberauth_cognito, "~> 0.1"}
```

Comment:
Is this `0.1` actually set anywhere? I think there's supposed to be a `:version` key in the `project/0` config. For example, see [here](https://github.com/mbta/gtfs_realtime_viz/blob/master/mix.exs) but I don't know if things have changed since then.

Reply:
Yeah, there's a `version` key set in `project` (we just don't have an `@version` for it like the example you linked to does).